README.md

```diff
@@ -1,5 +1,4 @@
 # Bitcoin Cash Specification Repository
 
-This repository contains markdown documentation that specifies the Bitcoin Cash cryptocurrency protocol.  However, this repository should not be directly used for normal browsing because this document uses some markdown extensions.
+This repository contains markdown documentation that specifies the Bitcoin Cash cryptocurrency protocol.
 
-Instead, read this documentation at [reference.cash](http://reference.cash)
```

Comment:
Hmm, we do still use markdown extensions, for example katex for math.  Base .md is very limiting.

Reply:
I recommend we keep the removed sentence and then add a link to documentation.cash and bitcoinprotocol.cash.